js/ad_manager_vast.js

```diff
@@ -790,23 +1141,22 @@ OO.Ads.manager(function(_, $) {
     }, this);
 
     /**
-     * Helper function that make sure the array is not empty.
+     * Helper function to remove empty items.
      * @private
      * @method Vast#filterEmpty
-     * @param {array} array An array that is the be checked if it is empty.
+     * @param {Array} array An array that is the be checked if it is empty
+     * @returns {Array} The filtered array.
```

Comment:
So according to: https://msdn.microsoft.com/en-us/library/7wkd9z69(v=vs.94).aspx
Array is a "composite" type and falls under the same category as Object, so I want to say that it is. 

Also, I did a quick double check on stackoverflow (because the JSDoc docs don't have many Array cases): http://stackoverflow.com/questions/14611995/how-to-specify-an-array-of-objects-as-a-parameter-or-return-value-in-jsdoc
We are using JSDoc3 so the use of Array does seem kosher.